src/docs/api/public.mdx

```diff
@@ -80,10 +24,16 @@ You **can** add attributes to a public endpoint.
 
 ## When to make an API public?
 
-Making an API public offers considerable benefits. A public API allows our customers to incorporate
-Sentry into their workflows. It also gives Sentry more visibility in our customers' organizations
-("Woah that's cool data, where'd it come from?"..."Sentry!"). Remember, however, making an API
-public means that once an endpoint is public, you can only add to it: You cannot make _any_ breaking
+Should I publish my API? The answer in most of the cases is **Yes**. As a developer facing company it is critical to make Sentry features available for developers to use in their various automations. Do not try to predict if an automation finds the API useful because we can't predict all the automations our clients might want to build. With that being said, there are 3 exceptions where we don't publish APIs:
```

Comment:
```suggestion
Should I publish my API? The answer in most cases is **Yes**. As a developer facing company it is critical to make Sentry features available for developers to use in their various automations. Do not try to predict if an automation finds the API useful because we can't predict all the automations our clients might want to build. With that being said, there are 3 exceptions where we don't publish APIs:
```